SeQuant/core/asy_cost.cpp

```diff
@@ -202,6 +215,14 @@ bool operator>(AsyCost const &lhs, AsyCost const &rhs) {
   return !(lhs < rhs || lhs == rhs);
 }
 
+bool operator<=(AsyCost const& lhs, AsyCost const& rhs) {
+  return lhs < rhs || lhs == rhs;
```

Comment:
```suggestion
  return !(lhs > rhs);
```
(tiny bit more efficient)